src/frontend/screens/Game/GameSubMenu/index.tsx

```diff
@@ -383,17 +348,15 @@ export default function GamesSubmenu({
             {t('submenu.protondb')}
           </button>
         )}
+        {onShowRequirements && (
+          <button
+            onClick={async () => onShowRequirements()}
+            className="link button is-text is-link"
+          >
+            {t('game.reuirements', 'Requirements')}
```

Comment:
I know there's a typo in `reuirements` but the translation key is already this, I didn't want to change it and break something

Reply:
not an issue if it is only the key.